mobile/screens/Training/AddTrainingLogScreen.tsx

```diff
@@ -74,25 +75,52 @@ export default function AddTrainingLogScreen(props: any) {
           />
 
           <Text style={styles.label}>Skills Displayed*</Text>
-          <SolidDropDown
-            items={{
-              "Post/Block": ServiceAnimalSkills.SKILL_POST_BLOCK,
-              "Lead/Follow": ServiceAnimalSkills.SKILL_LEAD_FOLLOW,
-              "Stay/Sit/Down": ServiceAnimalSkills.SKILL_STAY_SIT_DOWN,
-              Touch: ServiceAnimalSkills.SKILL_TOUCH,
-              Tuck: ServiceAnimalSkills.SKILL_TUCK,
-              Heel: ServiceAnimalSkills.SKILL_HEEL,
-            }}
-            isMultiselect={true}
-            placeholder="Select to Add"
-            callbackFunction={(
-              values: string[] | string,
-              keys: string[] | string
-            ) => {
-              setSkillValuesSelected([...(values as string[])]);
-              setSkillKeysSelected([...(keys as string[])]);
-            }}
-          />
+          {props.route.params.hoursCompleted > 800
```

Comment:
Move `items` up out of the JSX and put a conditional there that adds the complex tasks to the object if over 800 hours to avoid duplicate code.

Reply:
✅